app/src/main/java/openfoodfacts/github/scrachx/openfood/fragments/AddProductOverviewFragment.java

```diff
@@ -112,6 +112,10 @@
     private static final String PARAM_PURCHASE = "add_purchase_places";
     private static final String PARAM_STORE = "add_stores";
     private static final String PARAM_COUNTRIES = "add_countries";
+    private static final String PARAM_OTHER_INFO = "other_information_fr";
```

Comment:
Can you use `other_information` instead, so that it can work accross languages ?
Ideally, we should retrieve it based on the locale, but we don't do it yet for fields like common_name or product_name.

Reply:
Okay, I will do it for all new four fields.